griptape/schemas/base_schema.py

```diff
@@ -52,28 +53,77 @@ def _get_field_for_type(cls, field_type: type) -> fields.Field | fields.Nested:
         Args:
             field_type: A field type.
         """
+        from enum import Enum
+
         from griptape.schemas.polymorphic_schema import PolymorphicSchema
 
         field_class, args, optional = cls._get_field_type_info(field_type)
 
-        # Resolve TypeVars to their bound type
+        # Handle TypeVars
```

Comment:
What was the reasoning for changing this comment?

Reply:
I changed the logic for handling TypeVars while working through tests, I will change this back. Just ended up accidentally deleting it.